CONTRIBUTING.md

```diff
@@ -148,11 +148,17 @@ Puppetfile and then run `bundle exec r10k puppetfile install`.
 ## Testing
```

Comment:
This doesn't need to be part of this ticket, but we should consider rewriting this section soon to be clearer that the tests you can run / the tools you need are dependent on the platform you're developing on. Specifically, LXD isn't available on macOS or Windows, so you can't run the LXD tests on those platforms.